### DISCUSSION TITLE
Is there a way to make this work with Bluetooth aswell?

### DISCUSSION THREAD

**Discussion by kreatoo** [pgj/freebsd-wifibox#49] [Ideas]
Hello, 
I want to use wifibox to switch to FreeBSD and i was wondering, does this work with Bluetooth aswell? If so, how can i make it work?
My wireless card is Intel AX210.
Thanks, really cool project!

**Comment by pgj** [comment]
I have not really explored this possibility, and currently there is no explicit support for it.  But this sounds like an interesting idea, and it is definitely worth some experimentation.  I am busy with other bugs and features for the moment, but I can leave this ticket open to gather the related findings.

**Comment by mkohanim** [comment]
I am very interested as well. BLE support on FreeBSD is very limited and limiting.
Thank you.
